content/news/XXXX-XX-XX-mixxx-2-3-0-release.md

```diff
@@ -0,0 +1,126 @@
+title: Mixxx 2.3.0 released
+authors: Jan Holthuis
+tags: 2.3, 2.3.0, release announcement
+status: draft
+comments: yes
+
+After a long waiting time, we are proud to announce the release of Mixxx 2.3.0, [download it!]({filename}/pages/download.md#stable)
+This release brings [hotcue colors & labels]({filename}/news/2020-08-25-new-in-2-3-hotcue-colors.md), import of [Serato](2021-02-08-new-in-2-3-serato-support.md) and [Rekordbox]({filename}/news/2020-07-20-new-in-2-3-rekordbox-support.md) libraries, [more accurate key detection]({filename}/news/2021-04-08-new-in-2-3-keyfinder.md), [intro/outro sections](2020-07-09-intro-outro-sections.md), a revamped LateNight skin, multithreaded analysis, [deck cloning]({filename}/news/2020-11-15-new-in-2-3-deck-clone.md), [track color support]({filename}/news/2020-10-24-new-in-2-3-track-colors.md) and more.
+
+![Screenshot of a Mixxx deck with the intro and outro cues and hotcue editing menu]({static}/images/news/screenshot-2.3-beta-deck.png){: style="max-height: 275px" }
+
+This has been a rather big release, with lots of useful changes, bug fixes and improvements.
```

Comment:
```suggestion
This is a rather big release, with lots of useful changes, bug fixes and improvements.
```
I know, for us it's now like 'case closed', 'done!' -- for users it's the start of something